components/vote/VoteImage.js

```diff
@@ -226,9 +226,11 @@ export default function VoteImage() {
     loadMore(true);
     showCards();
 
-    if(window.localStorage.getItem("isReloadedFirstTimeAfterLogin")=="true"){
-      notify({ text: "You’re on the Lens Testnet",duration:10000});
-      window.localStorage.removeItem('isReloadedFirstTimeAfterLogin');
+    if (
+      window.localStorage.getItem("isReloadedFirstTimeAfterLogin") == "true"
```

Comment:
boolean variables shouldn't be in string

Reply:
will be resolved in next PR